### DISCUSSION TITLE
[要望] カスタムバンドルの追加

### DISCUSSION THREAD

**Discussion by anminmakura** [TUSB/TheUnusualSkyBlock#498] [Idea]
### アイデアのカテゴリー

アイテム・TUSBメモリー

## 機能
カスタムバンドルに任意のアイテムを収納することで
そのアイテムを登録され、<アイテム名>・バンドルと表記されるようになります。

その後、登録されたアイテムがインベントリーに入手した際、
自動でスタックするまで補完するようになります。

### アイデアの要望理由

以前からTUSBでのチェストのアイテムの整頓時間を
減らせないものかと難儀していたので
このようなアイテムがあれば少しでも整頓時間が減らせないかと考案しました。

**Comment by waxsd100** [comment]
> [!NOTE]
> 要望ラベルは実装必須の**機能追加**と実装必須ではない**アイデア**が混在しているため非推奨ラベルです。
> 機能追加(実装確定)なら`機能追加`ラベル
> アイデア(要望や実装未確定)なら`アイデア`ラベル
> を追加し、`要望`ラベルは削除してください。
